tests/test_ayab_communication.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+# This file is part of AYAB.
+#
+#    AYAB is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    AYAB is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with AYAB.  If not, see <http://www.gnu.org/licenses/>.
+#
+#    Copyright 2015 Shiluka Dharmasena, Sebastian Oliva, Christian Obersteiner, Andreas M�ller
+#    https://bitbucket.org/chris007de/ayab-apparat/
+
+import pytest
+import serial
+import unittest
+from knitlib.plugins.ayab_plugin.ayab_communication import AyabCommunication
+from mock import patch
+
+
+class TestCommunication(unittest.TestCase):
+
+  def setUp(self):
+    self.dummy_serial = serial.serial_for_url("loop://logging=debug")
+    self.comm_dummy = AyabCommunication(self.dummy_serial)
+
+  """Test on closing serial port communication."""
```

Comment:
Doc strings go inside the methods. http://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html